Examples/VimeoUpload-iOS/VimeoUpload-iOS/ViewControllers/VideoSettingsViewController.swift

```diff
@@ -285,8 +285,20 @@ class VideoSettingsViewController: UIViewController, UITextFieldDelegate
     
     private func applyVideoSettings()
     {
-        let videoUri = self.uploadTicket!.video!.uri!
-        let videoSettings = self.videoSettings!
+        guard let videoUri = self.uploadTicket?.video?.uri, let videoSettings = self.videoSettings else
+        {
+            let alertController = UIAlertController(
+                title: "Cannot Upload Video",
+                message: "The video object in `uploadTicket` does not exist!",
```

Comment:
Should this somehow tell the user that if they are trying to upload using new-upload target (which uses 4 step upload), it won't work? @supperspidey